src/libraries/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSourceActivity.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace System.Diagnostics
 {
-    public partial class Activity
+    public partial class Activity : IDisposable
```

Comment:
What does this mean for all activity use today where it's not being disposed? Are we introducing leaks?

Reply:
We are not introducing a leak. This is added for simplifying the pattern of using Activity. We are not introducing any leaks. I am going to add some code samples in the issue we'll use for the design review.